#CMake does not find -lXcursor -lXfixes -lXinerama libraries needed by FLTK if libX11 and indpendent libs used instead of X11
#Oct 14th 2016 by B. Hajgato (Free University Brussels - VUB) 
--- tcblab-relion2-beta-a6225608e270/CMakeLists.txt.orig	2016-09-28 12:40:23.000000000 +0200
+++ tcblab-relion2-beta-a6225608e270/CMakeLists.txt	2016-10-14 16:22:52.627892914 +0200
@@ -188,6 +188,7 @@
             FIND_PACKAGE(FLTK)
 		    if(FLTK_FOUND)
 				message(STATUS "X11 and FLTK were found")
+                                list(APPEND FLTK_LIBRARIES ${FLTK_LIBRARIES} "-lXcursor;-lXfixes;-lXinerama")
 				message(STATUS "FLTK_LIBRARIES: ${FLTK_LIBRARIES}")
 		    else()
 		        message(STATUS "FLTK was NOT found")
